### ISSUE TITLE
Datatable: loading slot not working in columns

### ISSUE THREAD

**Issue by devmadhavv** [primefaces/primevue#5505]
### Describe the bug

I wanted to use skeleton component for each columns for showing loading in the primevue Datatable. Couldn't able to customise the loading slot for each columns, instead I am getting a big circular progress indicator which does not suits my design. 

I don't want to use virtualScroller (as custom loading template was working only for that)

### Reproducer

https://stackblitz.com/edit/zbds1u?file=src%2FApp.vue

### PrimeVue version

3.48.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Nuxt

### Steps to reproduce the behavior

1. Paginate the datatable
2. Skeleton does not appear in columns

### Expected behavior

I want it to work like [virtual lazy datatable](https://primevue.org/datatable/#lazy_virtualscroll), the only difference is that I am using pagination and not virtual scrolling behaviour. On pagination I should be able to see skeletons whenever the loading state is true

**Comment by ikasianiuk** [comment]
any news on this one? i have the same issue. Maybe some workaround?

**Comment by devmadhavv** [comment]
Yeah I got the workaround for my use-case. I am not using the loading of data-table and just relying on my local states of api calls. For pending state I just conditionally rendering my Skeleton components and hiding my rows
